plugins/woocommerce/src/Admin/PageController.php

```diff
@@ -125,7 +125,7 @@ public function determine_current_page() {
 		}
 
 		$current_query = wp_parse_url( $current_url, PHP_URL_QUERY );
-		parse_str( $current_query, $current_pieces );
+		parse_str( $current_query ? $current_query : '', $current_pieces );
```

Comment:
This works, though we could optionally consider (a little shorter, possibly clearer):

```suggestion
		parse_str( (string) $current_query, $current_pieces );
```

Reply:
Yeah, good point and worked in all versions. https://3v4l.org/Y5vvZ